.github/workflows/angular.yml

```diff
@@ -71,7 +71,6 @@ jobs:
                     - ngx-gradle-psql-es-noi18n-mapsid
                     - ngx-gradle-mariadb-oauth2-infinispan
                     - ngx-gradle-mongodb-kafka-cucumber
-                    - ngx-gradle-yarn-h2disk-ws-nocache
```

Comment:
Don't remove this. It test more than yarn. Just amend config to use npm

Reply:
You are right, I've renamed this one by removing Yarn reference but I wonder why i've also removed this entry. Probably a mistake.

Reply:
In the end, I will review all builds and delete the less used options. We have lot of problem with GitHub Actions as we want to test too much options in our PR.

Reply:
If you are ok @pascalgrimaud I suggest to let this as is and reduce builds in another PR or would like me to already remove from this one?

Reply:
Yes, I'll reduce builds in another PR